framework/math/quadratures/angular/sldfe_sq_quadrature.h

```diff
@@ -196,38 +196,38 @@ struct SimplifiedLDFESQ::FUNCTION_WEIGHT_FROM_RHO
     // Init RHS
     for (int i = 0; i < 4; ++i)
     {
-      rhs[i] = std::vector<double>(4);
-      c_coeffs[i] = std::vector<double>(4);
+      rhs[i] = Vector<double>(4);
+      c_coeffs[i] = Vector<double>(4, 0.);
       for (int j = 0; j < 4; ++j)
-        rhs[i][i] = 1.0;
+        rhs[i](i) = 1.0;
```

Comment:
Arghhh. `[i](i)` because it's std of array of a Vector (but same would be with an std of vec of a Vector). For sure, I need to get used to this